thyrel-web/src/components/room/start/SentenceForm.tsx

```diff
@@ -29,7 +31,9 @@ export default function StartForm() {
   const [sentence, setSentence] = React.useState(currentElement?.text || '');
   const [loading, setLoading] = React.useState(false);
   const isEditing = Boolean(!currentElement?.finishAt);
-  const defaultSentence = useRandomSentence();
+  const { room } = useRoomContext();
+  const isOneWord = room?.mode === RoomMode.OneWord;
+  const defaultSentence = useRandomSentence(isOneWord);
```

Comment:
```suggestion
  const defaultSentence = useRandomSentence(room?.mode === RoomMode.OneWord);
```
c'est ce que j'aurais fais mais ça dérange pas du tout comme tu as fais.